packages/components/molecules/f-rating/src/components/Rating.vue

```diff
@@ -165,6 +164,20 @@ export default {
          */
         shouldDisplayUserOwnRating () {
             return this.isUserRating && this.hasRatingAvailable;
+        },
+
+        /**
+         * Returns the star rating is within the valid range or 0 otherwise
+         *
+         * @returns {boolean}
+         */
+        starRatingWithValidation () {
```

Comment:
I'd probably call this:

```suggestion
        validatedStarRating () {
```

(Would need to update the other references to this throughout the PR)

Reply:
cool, have renamed @ashleynolan , thanks for the suggestion